src/plugins/cosmos/components/StakedRow/StakedRow.tsx

```diff
@@ -0,0 +1,34 @@
+import { Flex, FlexProps } from '@chakra-ui/layout'
+import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
+import { Amount } from 'components/Amount/Amount'
+import { Text } from 'components/Text'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+type StakedRowProps = {
+  assetSymbol: string
+  cryptoStakedAmount: BigNumber
+  fiatRate: BigNumber
+  apr: BigNumber
+}
+export const StakedRow = ({
+  cryptoStakedAmount,
+  fiatRate,
+  assetSymbol,
+  apr,
+  ...styleProps
+}: StakedRowProps & FlexProps) => (
+  <Flex width='100%' justifyContent='space-between' {...styleProps}>
```

Comment:
You can't remove it because of https://github.com/shapeshift/web/pull/1047/files#diff-387e69970e8ca87c7aaf797ad5db351ba9c808d7e35b5738a6b67e867681a2c4R31

It's fine